QualificationChecker/Controllers/UtilityClass.cs

```diff
@@ -0,0 +1,122 @@
+﻿using QualificationChecker.Models;
+
+namespace QualificationChecker.Controllers
+{
+    public abstract class UtilityClass
+    {
+        protected UtilityClass(Candidate candidate, Organization organization)
+        {
+            Candidate = candidate;
+            Organization = organization;
+        }
+
+        public Candidate Candidate { get; private set; }
+
+        public Organization Organization { get; private set; }
+
+        public Dictionary<int, List<OrgQuestion>> OrgAnswersDictionary
+        {
+            get
+            {
+                return Organization.OrgQuestions
+                                    .GroupBy(q => q.PositionId)
+                                    .ToDictionary(group => group.Key, group => group.ToList());
+            }
+        }
+
+    }
+    // fake incoming query call that an http request
+    public class MatchRequest
+    {
+        public Organization Organization { get; set; }
+        public List<Candidate> Candidates { get; set; }
+    }
+
+    public class CandidateUtilities : UtilityClass
+    {
+        public CandidateUtilities(Candidate candidate, Organization organization) : base(candidate, organization)
+        {
+        }
+
+        // Method to tell if a Canidate is intrested in the position
+        public bool IsInterested()
+        {
+            if (Candidate.InterestedPositionIds == null || !Candidate.InterestedPositionIds.Any())
+            {
+                return false;
+            }
+
+            return Candidate.InterestedPositionIds.Any(candidatePositionId =>
+                Organization.OrgQuestions.Any(OrgQuestion => OrgQuestion.PositionId == candidatePositionId));
+        }
+        // Method to make sure they meet the age requirement
+        public bool MeetsAgeRequirement()
+        {
+            return Candidate.Age >= Organization.MinAgeRequirement;
+        }
+
+        // Method to return if the answers match
+        private bool MatchAnswers(bool candidateAnswers, bool organizationAnswers)
```

Comment:
Should these two variables be plural?  I would expect this to be comparing a single candidate answer with a single organization answer.